charts/consul/templates/client-snapshot-agent-deployment.yaml

```
@@ -1,5 +1,6 @@
 {{- if (or (and (ne (.Values.client.enabled | toString) "-") .Values.client.enabled) (and (eq (.Values.client.enabled | toString) "-") .Values.global.enabled)) }}
 {{- if .Values.client.snapshotAgent.enabled }}
+{{- if .Values.server.enterpriseLicense }}{{ fail ".Values.server.enterpriseLicense has been moved to .Values.global.enterpriseLicense" }}{{ end -}}
```

```suggestion
{{- if .Values.server.enterpriseLicense }}{{ fail "server.enterpriseLicense has been moved to global.enterpriseLicense" }}{{ end -}}
```

Should be an error the user can understand. The don't know what .Values is.